packages/mangrove/src/kandel.ts

```diff
@@ -48,21 +58,6 @@ export interface AllAsks {
   type: "AllAsks";
 }
 
-export interface SetParams {
```

Comment:
Is it really not possible for param setting events to occur separately in consequent transactions?